internal/generator/vector/output/syslog/rfc3164_with_app_name_setting.toml

```diff
@@ -0,0 +1,24 @@
+[transforms.example_parse_encoding]
+type = "remap"
+inputs = ["application"]
+source = '''
+. = merge(., parse_json!(string!(.message))) ?? .
+
+.tag = to_string!(.systemd.u.SYSLOG_IDENTIFIER[.systemd.t.PID] || .systemd.u.SYSLOG_IDENTIFIER || "-")
```

Comment:
I think we want the fallback to be empty spaces for all of these.  Fluentd used "-" but I do not see where dash is anything in the spec